tests/framework/installer/install_data.go

```diff
@@ -222,11 +222,11 @@ metadata:
   name: rook-tools
   namespace: ` + namespace + `
 spec:
+  dnsPolicy: ClusterFirstWithHostNet
```

Comment:
have you rebased recently? I believe changes to this file are already upstream. The changes must be the same since it isn't complaining about a conflict

Reply:
I think i didnt fetch the latest before rebasing.